### PR TITLE
Bug/wa 336 qgis not closing dcv session

### PR DESCRIPTION
**Pull request by van-go**
I updated the apps.json to use the interactive template. 

Question: Should I delete the tapisjob_app.sh from the qgis-s3 folder?

### CODE REVIEW

**Comment by chandra-tacc** [comment]
> Question: Should I delete the tapisjob_app.sh from the qgis-s3 folder?

Yes. If we want to revert, there is git log.

**Review by chandra-tacc — APPROVED** [review]
LGTM.
Tested at https://www.designsafe-ci.org/rw/workspace/qgis-s3?appVersion=3.34.4

File: applications/visit-stampede3/app.json [hunk 1]
```diff
@@ -87,7 +87,7 @@
       "portalName: DesignSafe"
     ],
     "notes": {
-      "icon": "Generic-Vis",
+      "icon": "Generic-Visualization",
```
Comment (chandra-tacc) [inline applications/visit-stampede3/app.json:90]: This is intentional change?

Comment (van-go) [reply applications/visit-stampede3/app.json:90]: Yes, it matches what's in the drop down menu in the App Listing Entry

**Pull request merged**